STYLEGUIDE.md

```diff
@@ -203,30 +213,9 @@ hash = { "one" => 1, "two" => 2, "three" => 3 }
 hash = { one: 1, two: 2, three: 3 }
 ```
 
-## Documentation
-
-Use [TomDoc](http://tomdoc.org) to the best of your ability. It's pretty sweet:
-
-``` ruby
-# Public: Duplicate some text an arbitrary number of times.
-#
-# text  - The String to be duplicated.
-# count - The Integer number of times to duplicate the text.
-#
-# Examples
-#
-#   multiplex("Tom", 4)
-#   # => "TomTomTomTom"
-#
-# Returns the duplicated String.
-def multiplex(text, count)
-  text * count
-end
-```
-
 ## Exceptions
 
-* Don't use exceptions for flow of control.
+* Don't use exceptions for flow control.
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Control_flow), the correct term is either 'flow of control' or 'control flow'. We use 'flow of control' later in the document, so I suggest switching to that for consistency.